### PR TITLE
New checkbox component .json

### PR DESCRIPTION
**Pull request by cifinan**
New .json file with tokens for the checkbox component.

Figma with spec for Checkbox component: https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4371%3A9978
-> Page 3.0 Checkbox/Slider... -> Checkboxes ‘Momentum Style’

### CODE REVIEW

File: components/checkbox.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "checkbox": {
+    "checked":{
+		"#normal":{
+			"background": "@theme-control-active-normal",
+			"text": "@theme-text-primary-normal",
+			"border": "@theme-control-active-normal"
```
Comment (paulwitty) [inline components/checkbox.json:7]: Can you do this as "border": "none" please?

File: components/checkbox.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		},
```
Comment (paulwitty) [inline components/checkbox.json:8]: What colour is the actual check icon? There should be a token for that.

Comment (cifinan) [reply components/checkbox.json:9]: The icon color is added. 👍

File: components/checkbox.json (hunk 1 above)
Comment (cifinan) [reply components/checkbox.json]: Yes. Updated. 👍

File: components/checkbox.json [hunk 3]
```diff
@@ -4,29 +4,33 @@
...
+			"border": "none",
+			"icon": "@theme-background-solid-primary-normal"
 		}
 	},
 	"unchecked":{
 		"#normal":{
 			"background": "@theme-background-secondary-normal",
 			"text": "@theme-text-primary-normal",
-			"border": "@theme-text-primary-normal"
+			"border": "@theme-text-primary-normal",
```
Comment (Coread) [inline components/checkbox.json:33]: trailing comma

File: components/checkbox.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
+{
+  "checkbox": {
```
Comment (paulwitty) [inline components/checkbox.json:2]: Can you add a "figma" property please linking to the relevant figma?

Comment (cifinan) [reply components/checkbox.json:2]: Ah yes, done.

**Review by paulwitty — APPROVED** [review]

**Pull request merged**